main.go

```diff
@@ -142,7 +142,9 @@ func main() {
 	blankController := controllers.NewBlankController(svc)
 	secured.GET("/getbtc", blankController.GetBtc)
 	secured.GET("/getpending", blankController.GetPending)
-	e.GET("/", blankController.Home)
+	e.GET("/", controllers.NewHomeController(svc).Home)
+	e.GET("/qr", controllers.NewHomeController(svc).QR)
```

Comment:
what's a good practice here actually? 
shouldn't this be one home controller instance?

Reply:
Yes, fixed. I feel like we can use a single controller for the two routes here.